addons/account/tests/test_account_payment.py

```diff
@@ -25,21 +25,25 @@ def setUpClass(cls):
             'acc_number': "0123456789",
             'partner_id': cls.partner_a.id,
             'acc_type': 'bank',
+            'sequence': 10,
```

Comment:
Hello ! I'm just taking a look at the account diff; is it still needed now that you no longer touches the `_order` on `res.partner.bank` ? :)

Reply:
Hi Julien, thanks for checking! You're right of course, it's no longer needed. I removed it and there's no more `account` changes in this PR now.